lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOGroupMembersDAOImpl.java

```diff
@@ -122,6 +128,14 @@
 		UserGroupUtils.copyDboToDto(dbos, members);
 		return members;
 	}
+	
+	@Override
+	public List<String> queryGroups(String principalId, List<String> groupIds) {
+		MapSqlParameterSource param = new MapSqlParameterSource();
```

Comment:
validate principalId is not null
if groupIds is null or empty then return empty list